README.md

```diff
@@ -37,7 +37,7 @@ It’s not a problem if you export your data to Spot Adventures and create an 
 
 I guess you need to cache the last used location yourself.
 
-spotparse.py
+Spotparse.py
```

Comment:
Hello! Thanks for the PR, but I think I'd like to keep this one lower-case, as it refers to an actual file name, which is lower case.